### PR TITLE
adds: essential netlify.toml config file

### LINKED ISSUE #1447: Build Gatsby and Next PR Previews
In #1444 I'm going to merge the Gatsby and Next branches together in `master` so we will have `src/frontend/gatsby` and `src/frontend/next`.  I'd like to find a way to build both frontends on every PR, while we are porting between the two.

In theory, we can do this with Vercel via [their monorepos support](https://vercel.com/blog/monorepos).  I've tried to understand how this works, but can't, so someone could experiement and do more reading.

Another option would be to do a mix of Netlify and Vercel, one for Gatsby and one for Next.

This is a short term solution, since we'll rip out the Gatsby one once Next is ready.

### PR DESCRIPTION
**Pull request by raygervais**
## Issue This PR Addresses
closes #1447 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [X] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

Adds configuration which allows testing and previews of NextJS to be viewed via Netlify

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: netlify.toml
Comment (humphd) [inline netlify.toml:6]: We probably need the api URL here from the env:

```
[build.environment]
  API_URL="https://dev.telescope.cdot.systems"
```

There might be others?

**Review by humphd — CHANGES_REQUESTED** [review]

File: netlify.toml
Comment (humphd) [reply netlify.toml:6]: I actually think that's probably it, based on the Vercel config: https://github.com/Seneca-CDOT/telescope/blob/master/now.json

Comment (raygervais) [reply netlify.toml:6]: Good thinking, will add to see it all work together now (fixed deployment settings).

**Review by humphd — APPROVED** [review]

**Comment by raygervais** [comment]
@humphd I'm going to do some more testing and see if I can get the preview to work before merging this.

**Comment by humphd** [comment]
Seeing this build error in various PRs: `Deploy directory 'src/frontend/next/out' does not exist`.  Thoughts?

**Comment by raygervais** [comment]
Resolved @humphd , this is good to merge once approved. Sorry for delay, work and life came up.

File: src/frontend/next/next.config.js
Comment (humphd) [inline src/frontend/next/next.config.js:2]: Why do we have to specify a build dir at all?

**Review by humphd — COMMENTED** [review]
Awesome.  One question.

File: src/frontend/next/next.config.js
Comment (raygervais) [reply src/frontend/next/next.config.js:2]: If not, outputs to `out` which doesn't follow our previous Gatsby semantics. I switched to `build` so that it's following similar semantics.

**Review by humphd — APPROVED** [review]

**Review by chrispinkney — APPROVED** [review]

**Pull request merged**
